packages/webapp/src/containers/Map/useDrawingManager.js

```diff
@@ -101,6 +106,9 @@ export default function useDrawingManager() {
         draggable: true,
       });
       redrawnMarker.setMap(map);
+      maps.event.addListener(redrawnMarker, 'dragend', () => {
```

Comment:
same as above I think this needs to be removed from the dom in the return statement of the useEffect.